atm.js

```diff
@@ -1,3 +1,75 @@
 $(document).ready(function(){
 // Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+var savings = {
```

Comment:
Organizing each account into an object -- nice! When we get into OOJS, you'll be able to refactor this even further so that `checking` and `savings` (and as many other accounts as you want) can be generating using a single object definition (or "constructor").